deployments/ansible/roles/collector/tasks/win_install_dotnet_auto_instrumentation.yml

```diff
@@ -0,0 +1,92 @@
+---
+- name: Get latest signalfx-dotnet-tracing release json
+  ansible.windows.win_uri:
+    url: "https://api.github.com/repos/signalfx/signalfx-dotnet-tracing/releases/latest"
+    return_content: yes
+    headers:
+      Authorization: "{{ signalfx_dotnet_auto_instrumentation_github_token }}"
```

Comment:
Does this not need a scheme? I wonder if setting conditionally would be helpful too